robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -152,6 +153,7 @@ public InstrumentationConfiguration build() {
           DoNotInstrument.class,
           Config.class,
           Transcript.class,
+          DependencyResolver.class,
```

Comment:
Apparently, there was some mess with classloading, and `InstrumentingClassLoader` tried instrumenting `CachedDependencyResolver`, which caused classloaders mismatch and the exception above.

Actually, I am somewhat surprised to see (at the first sight, arbitrary) internal Robolectric classes listed here. I don't have a deep understanding of the `InstrumentingClassLoader`, but shouldn't it only load android-specific classes? Is there really a need to explicitly specify Robolectric classes to exclude? Or am I missing something?